examples/jsm/nodes/materials/Materials.js

```diff
@@ -13,28 +12,6 @@ export {
 	PointsNodeMaterial
 };
 
-const materialLib = {
-	NodeMaterial,
-	LineBasicNodeMaterial,
-	MeshBasicNodeMaterial,
-	MeshStandardNodeMaterial,
-	PointsNodeMaterial
-};
-
-const fromTypeFunction = Material.fromType;
-
-Material.fromType = function ( type ) {
-
-	if ( materialLib[ type ] !== undefined ) {
-
-		return new materialLib[ type ]();
-
-	}
-
-	return fromTypeFunction.call( this, type );
-
-};
-
 NodeMaterial.fromMaterial = function ( material ) {
```

Comment:
Please also fix this function, it was using materialLib.

Reply:
good catch!